### ISSUE TITLE
Path based policy failed to apply, give invalid value for DNS-1123 subdomain error

### ISSUE THREAD

**Issue by psychomelet** [pomerium/pomerium#607]
Policy:
```
- from: https://test.yogorobot.com/f
  to: http://test.svc.cluster.local/f
  cors_allow_preflight: true
  allow_public_unauthenticated_access: true
```

Error

> Error: UPGRADE FAILED: cannot patch "yogo-pomerium" with kind Ingress: Ingress.extensions "yogo-pomerium" is invalid: [spec.rules[4].host: Invalid value: "test.yogorobot.com/f": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*'), spec.tls[7].hosts: Invalid value: "test.yogorobot.com/f": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')]

**Comment by branchmispredictor** [comment]
As far as I can tell, Pomerium does not support path-based policies. See #378 and #591. In your usecase, can you just use the following (since both from and too end with `/f`)?

```yaml
- from: https://test.yogorobot.com/
  to: http://test.svc.cluster.local/
  cors_allow_preflight: true
  allow_public_unauthenticated_access: true
```

**Comment by branchmispredictor** [comment]
@psychomelet, PR #615 to implement path-based route matching was just merged. Does this fix your usecase (provided you update your config to use the path-based routing)? You can use the `pomerium/pomerium:master` docker image to test it.

**Comment by psychomelet** [comment]
it works great! thanks

**Comment by desimone** [comment]
Great -- thanks @psychomelet @branchmispredictor !

Fixed with #615
